src/MediaWiki/Jobs/ChangePropagationDispatchJob.php

```diff
@@ -190,8 +190,8 @@ public static function getPendingJobsCount( DIWikiPage $subject ) {
 	public function run() {
 		$subject = DIWikiPage::newFromTitle( $this->getTitle() );
 
-		if ( $this->hasParameter( 'dataFile' ) ) {
-			return $this->dispatchFromFile( $subject, $this->getParameter( 'dataFile' ) );
+		if ( $this->hasParameter( 'data' ) ) {
```

Comment:
This looks like a breaking change that needs documenting in https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/docs/releasenotes/RELEASE-NOTES-5.0.0.md

Reply:
Thanks, done.